dir.props

```diff
@@ -69,13 +69,15 @@
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == ''">false</Optimize>
     <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
-    <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
+    <DefineConstants Condition="'$(MSBuildProjectExtension)' != '.vbproj'">$(DefineConstants);DEBUG;TRACE</DefineConstants>
```

Comment:
I would prefer if we didn't need to do this in everyone's dir.props and we only had one way to have DefineConstances which is with semi-colon's and then all in one place you translate them to comma delimited.